diskutil.py

```diff
@@ -280,6 +280,9 @@ def getDiskDeviceSize(dev):
         return int(__readOneLineFile__("/sys/block/%s/device/block/size" % dev))
     elif os.path.exists("/sys/block/%s/size" % dev):
         return int(__readOneLineFile__("/sys/block/%s/size" % dev))
+    else:
+        raise Exception("%s not found as %s or %s" % (dev, "/sys/block/%s/device/block/size",
```

Comment:
I think this will end up raising as e.g. `sda not found as /sys/block/%s/device/block/size or /sys/block/%s/size`, i.e. the `%s` characters will come out in the exception text, which doesn't seem quite what you'd expect?

If we don't need this for xs8 (i.e. we can rely on py3 functionality), could we use a format string to make it easier, at which point you'd just have e.g. `raise Exception(f"{dev} not found as /sys/block/{dev}/device/block/size or /sys/block/{dev}/size")`?

Reply:
We are trying to keep Python 2/3 compatibility, or at least have `master` and `release/xs8` branches as similar as possible. Maybe a more generic and easier `"Unable to get block size of %s device" % dev` would be enough, also not requiring to be updated as new potential virtual files are needed to be checked.

Reply:
moving the full paths inside the format string should be good for now.  At least using `str.format` and `{0}` will allow to avoid repeating `dev`